src/pages/IndividualArticle.jsx

```diff
@@ -1,16 +1,18 @@
 import { useEffect, useState } from "react";
 import { useParams } from 'react-router-dom';
 import axios from "axios"
+import CommentTile from "../components/CommentTile";
 
 export default function IndividualArticle () {
     const {article_id} = useParams()
 
+
     const newsApi = axios.create({
         baseURL: 'https://nc-news-p6kr.onrender.com/api/articles'
     })
 
```

Comment:
Again please put this in an API file so you only need to create one baseURL for **all** your API calls across the frontend